### PR TITLE
[55362] Updated Tigrinya texts

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

**Review by pavshr — APPROVED** [review]

File: NDB.Covid19/NDB.Covid19/Locales/ti.json [hunk 1]
```diff
@@ -248,7 +248,7 @@
   "REGISTER_ERROR_FETCH_FHI_DATA_DESCRIPTION": "ናይ ዝቐረበ እዋን ዳታ/ጸብጻብ ምምጻእ ዝተገብረ ፈተነ ኣይተዓወተን። ብኽብረትካ ጸኒሕካ ፈትን።",
   "REGISTER_QUESTIONAIRE_HEADER": "እቶም ዝድለዩ ሰባት መታን ክሕበሩ፡ ልክዕ መዓስ ከም ዝሓመምካ Smittestopp ክፈልጥ ኣድላዪ ኢዩ።",
   "REGISTER_QUESTIONAIRE_SYMPTOMONSET_TEXT": "ምልክታት ናይ ኮቪድ-19 ኣሎካ ድዩ ወይ ነይሩካ ድዩ?  ",
-  "REGISTER_QUESTIONAIRE_SYMPTOMONSET_ANSWER_YES": "እወ፣ ቅድሚ . .  ኣቢሉ",
+  "REGISTER_QUESTIONAIRE_SYMPTOMONSET_ANSWER_YES": "እወ፣ ካብ ዕለት …… ኣቢሉ",
```
Comment (mmn-nc) [inline NDB.Covid19/NDB.Covid19/Locales/ti.json:251]: Only 3 dots should be used here.

File: NDB.Covid19/NDB.Covid19/Locales/ti.json [hunk 2]
```diff
@@ -75,23 +75,23 @@
...
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_1_CONTENT": "እቲ ኣፕ ንጡፍ ኣብ ዝህልወሉ ግዜ፡ ኣብ ኖርወይን፡ ኣብ ብርክት ዝበላ ሃገራት EU ዝርግሐ ኮቪድ-19 ደው ክብል ኣበርክቶ ትገብር ኣለኻ። ኣብ ዝኾነ ግዜ ኣብቲ መጀመርያ ገጽ  ናይዚ ኣፕ ዘሎ 'ኣስርሓዮ' ወይ 'ዕጸዎ' ዝብል መልጐም ብምጥዋቕ ምዝገባ ክትከፍቶን ከተጥፍኦን ትኽእል ኢኻ። ናትካ ኣፕ ዘይንጡፍ እንተ'ልዩ ግን - ብቀይሕ ድሕረ ባይታ ምልክት  ዝረአ - ቴለፎንካ ምምዝጋብ የቛርጽ። ",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_1_CONTENT2": "ምስ ተሰማማዕካን ን Bluetooth ምስ ኣፍቀድካን እቲ ኣፕ ብቐጻሊ ክሰርሕ እዩ። ነቲ ስምምዕ ኣብ ትሕቲ 'ውዕልን ብሕታውነትን '  ኣብ ዝብል ኣቲኻ ክትስሕብ ትኽእል።",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_TITLE": "ውጽኢት መርመራኻ ምክፋል",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT": "ኣወንታዊ ውጽኢት መርመራ እንተ ተነጊሩካ፡ ብወለንታኻ ኣብቲ መጀመርያ ገጽ ናይዚ ኣፕ ኣቲኻ፡ ‘ኣወንታዊ መርመራ (postiv) ጸኒሑካ ድዩ?’ ኣብ ዝብል ብምጽቃጥ ሓበሬታ ኵነታትካ ከተካፍል ትኽእል። ካልኦት ተጠቀምቲ መን ምዃንካ ክርእዩ ኣይክእሉን እዮም። ብመንገዲ ID-porten ክትኣቱ ኣለካ። ምስ ብርክት ዝበላ ሃገራት EU መንነትካ ከይገለጽካ ሓበሬታኻ ከምዝካፈል ከተፍቕድ ትኽእል ኢኻ። ",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT": "ኣወንታዊ ውጽኢት መርመራ እንተ ተነጊሩካ፡ ብወለንታኻ ኣብቲ መጀመርያ ገጽ ናይዚ ኣፕ ኣቲኻ፡ ‘ኣወንታዊ መርመራ (positive) ጸኒሑካ ድዩ?’ ኣብ ዝብል ብምጽቃጥ ሓበሬታ ኵነታትካ ከተካፍል ትኽእል። ካልኦት ተጠቀምቲ መን ምዃንካ ክርእዩ ኣይክእሉን እዮም። ብመንገዲ ID-porten ክትኣቱ ኣለካ። ምስ ብርክት ዝበላ ሃገራት EU መንነትካ ከይገለጽካ ሓበሬታኻ ከምዝካፈል ክተፍቅድ ትኽእል ኢኻ። ",
```
Comment (mmn-nc) [inline NDB.Covid19/NDB.Covid19/Locales/ti.json:83]: Should be (positiv)

File: NDB.Covid19/NDB.Covid19/Locales/ti.json [hunk 3]
```diff
@@ -75,23 +75,23 @@
...
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT2": "ብዛዕባ ኵነታትካ ሓበሬታ ምስ ኣካፈልካ፡ ካልኦት ተጠቀምቲ ኣብ ጥቓ ሓደ ብኮቪድ -19 ዝተለኸፈ ሰብ  ከም ዝነበሩ ዝገልጽ መልእኽቲ ይበጽሖም።   ",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT3": "መልእኽትታት ኣብ መጀመርያ ገጽ ናይዚ ኣፕ  ክትሪኦም ትኽእል። ሓደስቲ መልእኽትታት መጺኦሙካ እንተ ሃሊዮም'ዉን ኣብኡ ትሪኦም። ",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT4": "Smittestopp ዘይንጡፍ እንተ'ልዩ መልእኽትታት ክትቅበል ኣይትኽእልን ኢኻ። ኣብቲ ኣፕ ኣቲኻ ኵነታትካ ከተካፍል ትኽእል ኢኻ፡ ግን እቲ ኣፕ ንጡፍ እንተ ዘይሃሊይ ናይ ID ምቅይያር ስለ ዘይግበር ኵነታትካ ትለዋወጠሎም ሰባት ኣይክህልዉን ኢዮም። ",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "ኣብ ብርክት ዝበላ ሃገራት EU ምዝርጋሕ ኮቪድ-19 ንምዕጋት ተሓባበር",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "እዚ ኣፕ ካብ ብርክት ዝበላ ናይ ኣውሮጳ ኮቪድ-19 ኣፕ ሓበረታ ይቕበል ። እዚ ዝግበር ምታን: ኣብ ጥቓ ሓደ ዝተለኽፈ ሰብ ተጠቓሚ ናይ ኤውሮጳ  ኮቪድ-19 ኣፕ እንተ ነርካ መልእኽቲ ንክትቅበል  ኢዩ። ከምኡ'ዉን ንስኻ ብኮቪድ-19 እንተ ተለኺፍካ፡ ንካሎት ኣብ ጥቓኻ ዝነበሩ ተጠቐምቲ ናይ ኣውሮጳ ኮቪድ-19 ኣፕ መልእኽቲ ክስደዶም ፍቓድ ክትህብ ትኽእል ኢኻ። እቲ ኣፕ ብዘይ ናትካ ፍቓድ ሓበሬታ ኣይካፈልን እዩ።  ",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT3": "መልእኽትታት ኣብ መጀመርያ ገጽ ናይዚ ኣፕ ትኽእል። ሓደስቲ መልእኽትታት መጺኦምኻ እንተ’ልዮም ኣብኡ ትርእዮም።",
```
Comment (mmn-nc) [inline NDB.Covid19/NDB.Covid19/Locales/ti.json:85]: ክትሪኦም should be replaced by ክትርእዮም and not just deleted

File: NDB.Covid19/NDB.Covid19/Locales/ti.json [hunk 4]
```diff
@@ -75,23 +75,23 @@
...
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT4": "Smittestopp ዘይንጡፍ እንተ'ልዩ መልእኽትታት ክትቅበል ኣይትኽእልን ኢኻ። ኣብቲ ኣፕ ኣቲኻ ኵነታትካ ከተካፍል ትኽእል ኢኻ፡ ግን እቲ ኣፕ ንጡፍ እንተ ዘየልዩ ናይ ID ምቅይያር ስለ ዘይግበር ኵነታትካ ሼር ትገብሮም ሰባት ኣይክህልዉን ኢዮም። ",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "ኣብ ብርክት ዝበላ ሃገራት EU ዝርጋሐ ኮቪድ-19 ንምዕጋት ተሓባበር",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "እዚ ኣፕ ካብ ብርክት ዝበላ ናይ ኣውሮጳ ሃገራት ኮቪድ-19 ኣፕ ሓበሬታ ይቕበል ። እዚ ዝግበር መታን: ኣብ ጥቓ ሓደ ዝተለኽፈ ሰብ ተጠቓሚ ናይ ኤውሮጳ  ኮቪድ-19 ኣፕ እንተ ነርካ መልእኽቲ ንክትቅበል  ኢዩ። ከምኡ'ዉን ንስኻ ብኮቪድ-19 እንተ ተለኺፍካ፡ ንካሎት ኣብ ጥቓኻ ዝነበሩ ተጠቐምቲ ናይ ኣውሮጳ ኮቪድ-19 ኣፕ መልእኽቲ ክስደዶም ፍቓድ ክትህብ ትኽእል ኢኻ። እቲ ኣፕ ብዘይ ናትካ ፍቓድ ሓበሬታ ኣይካፈልን እዩ።  ",
```
Comment (mmn-nc) [inline NDB.Covid19/NDB.Covid19/Locales/ti.json:88]: ተጠቓሚ should be replaced with ተጠቃሚ

**Pull request merged**